scss/_variables.scss

```diff
@@ -1245,6 +1245,7 @@ $card-cap-padding-y:                $card-spacer-y * .5 !default;
 $card-cap-padding-x:                $card-spacer-x !default;
 $card-cap-bg:                       rgba($black, .03) !default;
 $card-cap-color:                    null !default;
+$card-title-color:                  null !default;
```

Comment:
Could you please regroup it with the other `$card-title-*` in the file?

Reply:
Fixed in https://github.com/twbs/bootstrap/pull/36979/commits/41419dfab6f4b22e0b1970700727ffa21525e611.